src/main/java/org/folio/bulkops/util/Constants.java

```diff
@@ -44,8 +44,8 @@ public class Constants {
   public static final String MSG_HOLDING_NO_CHANGE_REQUIRED_UNSUPPRESSED_ITEMS_UPDATED = "No change in value for holdings record required, associated unsuppressed item(s) have been updated.";
   public static final String MSG_HOLDING_NO_CHANGE_REQUIRED_SUPPRESSED_ITEMS_UPDATED = "No change in value for holdings record required, associated suppressed item(s) have been updated.";
   public static final String STAFF_ONLY = "(staff only)";
-  public static final String ADMINISTRATIVE_NOTES = "Administrative Notes";
-  public static final String ADMINISTRATIVE_NOTE = "Administrative Note";
+  public static final String CSV_COLUMN_NAME_ADMINISTRATIVE_NOTE = "Administrative note";
```

Comment:
are you sure you need 2 the same strings?

Reply:
removed